util/docker/build.sh

```diff
@@ -54,7 +54,7 @@ fi
 
 # Build and push the image.
 image="$1"/"$2"
-sudo docker build \
+sudo docker buildx build \
```

Comment:
Is the duplication here intended?

Reply:
[Yes](https://docs.docker.com/engine/reference/commandline/buildx_build/).